Python/scoreP2B.py

```diff
@@ -26,16 +29,7 @@ def matchFeatureInputFile(truthFile, testDir):
     return testPathCandidates[0]
 
 
-
-def scoreP2BImage(truthPath, testPath):
-    truthImage = loadSegmentationImage(truthPath)
-    testImage = loadSegmentationImage(testPath)
-
-    if testImage.shape[0:2] != truthImage.shape[0:2]:
-        raise ScoreException('Image %s has dimensions %s; expected %s.' %
-                             (os.path.basename(testPath), testImage.shape[0:2],
-                              truthImage.shape[0:2]))
-
+def scoreP2BConcatenatedImages(truthImage, testImage):
     truthBinaryImage = (truthImage > 128)
```

Comment:
You lost this piece of logic. When loading, we make the guarantee that the image contains only the values `0` or `255`.

The biggest thing is that using the `>` operator returns an array of `numpy.bool_`. So, in the old code, `truthBinaryImage` is a bool array.